extensions/azurecore/src/account-provider/azureAccountProvider.ts

```diff
@@ -192,6 +228,11 @@ export class AzureAccountProvider implements azdata.AccountProvider, vscode.Disp
 		}
 	}
 
+	private isValidToken(accessToken: Token): boolean {
```

Comment:
I'd suggest adding a doc comment here, it's not clear from the name alone what "valid" means in this context (specifically the fact that it checks that it's a non-expired token)